tools/convertTranslations.js

```diff
@@ -21,14 +21,14 @@ function parseAndroid(data) {
   for (let line of lines) {
     let kv = line.match(rgxKeyValue);
     if (kv != null) {
-      result.parsed.push([kv[1], kv[2].replace(/&quot;/g, '"').
+      result.parsed.push([kv[1], kv[2].replace(/&quot;/g, '\"').
       replace(/&lt;/g, '<').
       replace(/&gt;/g, '>').
       replace(/&amp;/g, '&').
-      replace(/\\t/g, '\t').
-      replace(/\\r/g, '\r').
-      replace(/\\n/g, '\n').
-      replace(/\\\\/g, '\\').
+      replace(/\t/g, '\t').
+      replace(/\r/g, '\r').
+      replace(/\n/g, '\n').
+      replace(/\\/g, '\\').
```

Comment:
again. this code has no effect. it replaces the regex with itself. You can remove it completely